src/libaktualizr/package_manager/ostreemanager.cc

```diff
@@ -39,9 +39,12 @@ static void aktualizr_progress_cb(OstreeAsyncProgress *progress, gpointer data)
                << " outstanding: " << outstanding_metadata_fetches;
```

Comment:
This looks great. I can't remember what all is possible to communicate through the events API. However, if possible, it might be nice to add something like:
~~~
    *mt->events_channel << std::make_shared<event::DownloadProgressReport>(mt->target, "Calculating...");
~~~

Reply:
@doanac Currently the event only has an integer to represent progress percentage. What is the advantage to just emitting a 0% progress event at this stage?

Just FYI for the bigger picture: this events infrastructure will probably change soon to something a bit cleaner and better-typed using Boost signals. When we do that work, we can take into account your request to have text status messages included. That doesn't seem unreasonable and might be useful elsewhere, too.

Reply:
0% is clever. works for me